clientlib/README.md

```diff
@@ -3,60 +3,64 @@
 This module will helps you wrap a confluent.Message as scheduler message.
 It will not send the message to the scheduler's topic, just preparing the message to be produced.
 
-* To create/update a schedule
+## To create or update a scheduled message
+
+```go
+package main
 
-```
 import(
     "github.com/etf1/kafka-message-scheduler/clientlib"
 )
 
-key := "video1"
-targetTopic := "target-topic"
-schedulerTopic := "scheduler-topic"
-scheduleID := "video1-schedule-online"
-later := time.Now().Add(1 * time.Hour).Unix()
+func main() {
+    targetTopic := "target-topic"
 
-msg := kafka.Message{
-    Key: bytes(key),
-    Timestamp:     now,
-    TimestampType: kafka.TimestampCreateTime,
-    Value:         bytes("some value"),
-    TopicPartition: kafka.TopicPartition{
-        Topic: &targetTopic,
-    },
-}
+    now := time.Now()
+    later := now.Add(1 * time.Hour).Unix()
 
-result, err := clientlib.Schedule(&msg, scheduleID, later, schedulerTopic)
+    msg := kafka.Message{
+        Key: []byte("video1"),
+        Timestamp:     now,
+        TimestampType: kafka.TimestampCreateTime,
+        Value:         []byte("some value"),
+        TopicPartition: kafka.TopicPartition{
+            Topic: &targetTopic,
+        },
+    }
 
-if err != nil {
-    log.Printf("unexpected error: %v", err)
-}
-```
+    schedulerMessage, err := clientlib.Schedule(&msg, "video1-schedule-online", later, "scheduler-topic")
+    if err != nil {
+        log.Printf("unexpected error: %v", err)
+    }
 
-*  To delete a schedule
+    producer, err := kafka.NewProducer(&kafka.ConfigMap{
```

Comment:
in the beginning i didn't want to include the producer part, because as i mentioned on my comment "**It will not send the message to the scheduler's topic,** just preparing the message to be produced."
but if it more clear with the producer part, i think it is a good point.

And perhaps you can also add the producer code in the delete schedule section just below ;) marked as HERE

Reply:
I can keep it with just your code if you want but I think it’s always better when a developer is able to copy/paste a code example and having it to work directly.

I will add the full code example for the delete function also.